packages/mux-player/test/errors.test.js

```diff
@@ -45,70 +45,70 @@ describe('errors', () => {
     assert(fired === true, 'the error handler was fired');
   });
 
-  it('default message for MediaError.MEDIA_ERR_ABORTED', async function () {
+  it('default message for MediaError.MEDIA_ERR_ABORTED', function () {
     const error = new MediaError(undefined, MediaError.MEDIA_ERR_ABORTED);
-    const { dialog } = await getErrorLogs(error);
+    const { dialog } = getErrorLogs(error);
     assert.equal(dialog.message, 'You aborted the media playback');
   });
 
-  it('default message for MediaError.MEDIA_ERR_NETWORK', async function () {
+  it('default message for MediaError.MEDIA_ERR_NETWORK', function () {
     const error = new MediaError(undefined, MediaError.MEDIA_ERR_NETWORK);
-    const { dialog } = await getErrorLogs(error);
+    const { dialog } = getErrorLogs(error);
     assert.equal(dialog.title, 'Network Error');
     assert.equal(dialog.message, 'A network error caused the media download to fail.');
   });
 
-  it('default message for MediaError.MEDIA_ERR_DECODE', async function () {
+  it('default message for MediaError.MEDIA_ERR_DECODE', function () {
     const error = new MediaError(undefined, MediaError.MEDIA_ERR_DECODE);
-    const { dialog } = await getErrorLogs(error);
+    const { dialog } = getErrorLogs(error);
     assert.equal(dialog.title, 'Media Error');
     assert.equal(
       dialog.message,
       'A media error caused playback to be aborted. The media could be corrupt or your browser does not support this format.'
     );
   });
 
-  it('default title for MediaError.MEDIA_ERR_SRC_NOT_SUPPORTED', async function () {
+  it('default title for MediaError.MEDIA_ERR_SRC_NOT_SUPPORTED', function () {
     const error = new MediaError(undefined, MediaError.MEDIA_ERR_SRC_NOT_SUPPORTED);
-    const { dialog } = await getErrorLogs(error);
+    const { dialog } = getErrorLogs(error);
     assert.equal(dialog.title, 'Source Not Supported');
   });
 
-  it('default title for any error', async function () {
+  it('default title for any error', function () {
     const error = new MediaError(undefined, 0);
-    const { dialog } = await getErrorLogs(error);
+    const { dialog } = getErrorLogs(error);
     assert.equal(dialog.title, 'Error');
   });
 
-  it('offline error logs', async function () {
+  it('offline error logs', function () {
     const error = new MediaError(undefined, MediaError.MEDIA_ERR_DECODE);
     const offline = true;
-    const { dialog } = await getErrorLogs(error, offline);
+    const { dialog } = getErrorLogs(error, offline);
     assert.equal(dialog.title, 'Your device appears to be offline');
     assert.equal(dialog.message, 'Check your internet connection and try reloading this video.');
   });
 
-  it('412 response code message for MediaError.MEDIA_ERR_NETWORK', async function () {
+  it('412 response code message for MediaError.MEDIA_ERR_NETWORK', function () {
```

Comment:
non-blocking: either in this PR or as a followup, we should write some more full integration tests that will actually request assets and internally throw these errors.

Reply:
yup good call! I've locally tested some of these assets